src/test/java/core/basesyntax/service/impl/FruitServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.FruitService;
+import core.basesyntax.strategy.AdditionStrategyImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.ReductionStrategyImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitServiceImplTest {
+    private static Map<Operation, OperationStrategy> operationStrategyMap;
+    private static FruitService fruitService;
+    private static List<TransactionDto> transactionDtoListCorrect;
+    private static List<TransactionDto> transactionDtoListIncorrectAmount;
+    private static List<TransactionDto> transactionDtoListIncorrectResult;
+    private static final TransactionDto testDto1 = new TransactionDto(Operation.fromString("b"),
```

Comment:
```suggestion
    private static final TransactionDto testDto1 = new TransactionDto(Operation.BALANCE,
```
and so on everywhere

Reply:
done